src/lua/functions/map/house_functions.cpp

```diff
@@ -90,6 +90,17 @@ int HouseFunctions::luaHouseGetRent(lua_State* L) {
 	return 1;
 }
 
+int HouseFunctions::luaHouseGetPrice(lua_State* L) {
+	// house:getPrice()
+	House* house = getUserdata<House>(L, 1);
+	if (house) {
+		lua_pushnumber(L, house->getPrice());
+	} else {
+		lua_pushnil(L);
```

Comment:
We can send a more specific error than nil, for example:

```suggestion
	if (!house) {
		reportErrorFunc("House not found");
		lua_pushnumber(L, 0);
		return1;
	}
	
	lua_pushnumber(L, house->getPrice());
```